### ISSUE TITLE
Quickstart non functional

### ISSUE THREAD

**Issue by clayote** [lllangWV/ParquetDB#1]
I ran the quickstart as written, and got 

```TypeError: ParquetDB.read() got an unexpected keyword argument 'table_name'```

I worked out that you'd renamed that argument to `dataset_name` and supplied it that way instead. That seemed to work...but got me an empty dataframe, not the contents of the employees table.

**Comment by clayote** [comment]
Fixed by #2

**Comment by lllangWV** [comment]
Hey,

Thanks so much for your interest in the package! I’m still in the development phase, so a lot of things are experimental at this stage. I'll look at your pull request as soon as I can. I am dealing with PC issues at the moment where I probably have to restore my computer to an earlier point.

Recently, I introduced a new class called `ParquetDatasetDB`, which operates on individual Parquet dataset. I then made `ParquetDB` wrap around `ParquetDatasetDB` to manage. I felt the term "table" didn’t quite capture what I’m doing. I have not updated the README.md to reflect this change, but I will get to it as soon as possible.

Looking ahead, I’m considering adding relational IDs between tables to make the system function more like a pseudo-relational database. To reflect this change, I’m thinking about renaming `ParquetDB` to `ParquetRelationalDB` and renaming `ParquetDatasetDB` to `ParquetDB`.


Any feedback would be appreciated for the project!


Best,  
Logan Lang
